.ci-scripts/x86-64-apple-darwin-release.bash

```diff
@@ -39,8 +39,8 @@ ASSET_DESCRIPTION="https://github.com/ponylang/ponyc"
 
 # Build pony installation
 echo "Building ponyc installation..."
-make install prefix=${BUILD_PREFIX} default_pic=${PIC} arch=${ARCH} \
```

Comment:
is there a reason that it no longer does `default_pic`?

Reply:
In a discussion a few months ago (I'll try to find it on Zulip), we agreed that everything would be compiled with `-pic` and statically linked.  So there's no need for the `default_pic` option.

Reply:
gotcha, is there a way to compile without -pic? if no, i think we should have an issue to eventually add support for not pic.

Reply:
Will do.